1-js-and-async-programming/1-javascript-fundamentals/1-time-difference.js

```diff
@@ -16,6 +16,17 @@ timeDifference(7200, 3400); // Expected output: "01:03:20"
 
 */
 
-const timeDifference = (a, b) => {};
+const formatTime = (num) => {
+  num = Math.floor(num);
+  return num < 10 ? `0${num}` : `${num}`;
+};
+
+const timeDifference = (a, b) => {
+  let diff = a > b ? a - b : b - a;
+  let hours = diff / 3600;
+  let mins = (diff % 3600) / 60;
+  let sec = diff % 60;
```

Comment:
if we won't overwrite our initial value, we don't need to use `let`
```suggestion
   const diff = Math.abs(a - b);
  const hours = Math.floor(diff / 3600);
  const mins = Math.floor((diff % 3600) / 60);
  const sec = diff % 60;
```